src/templates/commit_page.handlebars

```diff
@@ -368,6 +362,18 @@
                             </span>
                         </li>
                         <li>
+                            <i class="section-group__item icon-check{{#is_false original.src_options.is_fanon}}-empty{{/is_false}}" data-field="is_fanon"/>
+                            <span class="check-txt">
+                                Is Fanon
```

Comment:
@russellholt Sorry for the late reply, @jdorweiler did describe what it really is. Pretty much every bigger wikia has that.